src/main/java/soconnect/commons/util/StringUtil.java

```diff
@@ -55,22 +54,54 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
     public static boolean containsKeywordsIgnoreCase(String sentence, String keywords) {
         requireAllNonNull(sentence, keywords);
 
-        List<String> preppedKeywords = Arrays.asList(keywords.toUpperCase().trim().split("\\s+"));
+        String preppedWord = keywords.trim();
+        checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
+        List<String> preppedKeywords = Arrays.asList(preppedWord.toUpperCase().trim().split("\\s+"));
         checkArgument(preppedKeywords.size() > 0, "Word parameter cannot be empty");
```

Comment:
What's the difference between this line and line 58? 
`checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");`

The two lines appear to achieve the same purpose?

Reply:
Oh ya it's because of how this test case is designed expects it to automatically throws exception if it's empty without splitting.
```
    @Test
    public void containsKeywordsIgnoreCase_emptyWord_throwsIllegalArgumentException() {
        assertThrows(IllegalArgumentException.class, "Word parameter cannot be empty", () ->
                StringUtil.containsKeywordsIgnoreCase("typical sentence", "  "));
    }
```